Lib/unittest/result.py

```diff
@@ -214,7 +214,7 @@ def _clean_tracebacks(self, exctype, value, tb, test):
 
             if value is not None:
                 for c in (value.__cause__, value.__context__):
-                    if c is not None:
```

Comment:
This doesn’t help with cycles involving more than one exception. The traceback module uses a ‘seen’ set to detect cycles.

Reply:
Addressed in c100409